qiita_db/metadata_template/test/test_sample_template.py

```diff
@@ -1342,20 +1343,22 @@ def test_update_numpy(self):
             'Sample2': {'bool_col': np.bool_(False),
                         'date_col': np.datetime64(datetime(2015, 8, 1))}
         }
-        metadata = pd.DataFrame.from_dict(metadata_dict, orient='index')
+        metadata = pd.DataFrame.from_dict(metadata_dict, orient='index',
+                                          dtype=str)
         st = npt.assert_warns(qdb.exceptions.QiitaDBWarning, ST.create,
                               metadata, self.new_study)
 
         metadata_dict['Sample2']['date_col'] = np.datetime64(
             datetime(2015, 9, 1))
         metadata_dict['Sample1']['bool_col'] = np.bool_(False)
-        metadata = pd.DataFrame.from_dict(metadata_dict, orient='index')
+        metadata = pd.DataFrame.from_dict(metadata_dict, orient='index',
+                                          dtype=str)
         npt.assert_warns(qdb.exceptions.QiitaDBWarning, st.update, metadata)
 
         sql = "SELECT * FROM qiita.sample_{0}".format(st.id)
         obs = self.conn_handler.execute_fetchall(sql)
-        exp = [['2.Sample1', False, datetime(2015, 9, 1)],
-               ['2.Sample2', False, datetime(2015, 9, 1)]]
+        exp = [['2.Sample1', 'false', '2015-09-01 00:00:00'],
+               ['2.Sample2', 'false', '2015-09-01 00:00:00']]
```

Comment:
`2015-09-01 00:00:00` for sample2 -- should this correspond to the date for sample2 on line 1344 above?
```
'Sample2': {'bool_col': np.bool_(False),
                          'date_col': np.datetime64(datetime(2015, 8, 1))}
```
Current one is 9/1/15 and the other is 8/1/15